client/src/components/Profile/ui/PrescreeningFeedback.tsx

```diff
@@ -3,65 +3,103 @@ import { StageInterviewDetailedFeedback } from '@common/models/profile';
 import { Rating } from '@client/components/Rating';
+import React from 'react';
 
 const { Text, Title } = Typography;
 
-export function PrescreeningFeedback({ feedback }: { feedback: StageInterviewDetailedFeedback['feedback'] }) {
-  const { steps } = feedback as { steps: Record<FeedbackStepId, InterviewFeedbackStepData> };
+const STYLES = {
+  feedbackItemWidth: '80%',
+  skillCommentWidth: '60%',
+};
 
-  const { theory, practice, english, decision, intro } = steps;
-  const isRejected = intro.values?.interviewResult === 'missed';
+const FEEDBACK_CONFIG = [
+  {
+    id: 'decision_redFlags',
+    label: 'Red flags',
+    getValue: (steps: Record<FeedbackStepId, InterviewFeedbackStepData>) => steps.decision.values?.redFlags,
+    isRejectedItem: false,
+  },
+  {
+    id: 'decision_comment',
+    label: 'Comment',
+    getValue: (steps: Record<FeedbackStepId, InterviewFeedbackStepData>) => steps.decision.values?.comment,
+    isRejectedItem: false,
+  },
+  {
+    id: 'english_certificate',
+    label: 'Certified level of English',
+    getValue: (steps: Record<FeedbackStepId, InterviewFeedbackStepData>) => steps.english.values?.englishCertificate,
+    isRejectedItem: false,
+  },
+  {
+    id: 'english_selfAssessment',
+    label: 'English level by interviewers opinion',
+    getValue: (steps: Record<FeedbackStepId, InterviewFeedbackStepData>) => steps.english.values?.selfAssessment,
+    isRejectedItem: false,
+  },
+  {
+    id: 'english_comment',
+    label: 'Where did the student learn English',
+    getValue: (steps: Record<FeedbackStepId, InterviewFeedbackStepData>) => steps.english.values?.comment,
+    isRejectedItem: false,
+  },
+  {
+    id: 'intro_comment',
+    label: 'Comment',
+    getValue: (steps: Record<FeedbackStepId, InterviewFeedbackStepData>) => steps.intro.values?.comment,
+    isRejectedItem: true,
+  },
+];
 
-  if (isRejected) {
+const FeedbackItem = ({
+  label,
+  value,
+  width = STYLES.feedbackItemWidth,
+}: {
+  label: string;
+  value: unknown;
+  width?: string;
+}) => {
+  if (typeof value === 'string' && value) {
     return (
-      <Space direction="vertical">
-        {intro.values?.comment && (
-          <Space>
-            <Text strong>Comment: </Text>
-            <Text>{intro.values?.comment as string} </Text>
-          </Space>
-        )}
+      <Space direction="vertical" style={{ width }}>
+        <Text strong>{label}: </Text>
+        <Text>{value}</Text>
       </Space>
     );
   }
+  return null;
+};
+
+export function PrescreeningFeedback({ feedback }: { feedback: StageInterviewDetailedFeedback['feedback'] }) {
+  const { steps } = feedback as { steps: Record<FeedbackStepId, InterviewFeedbackStepData> };
+  const { theory, practice } = steps;
+
+  const isRejected = steps.intro.values?.interviewResult === 'missed';
+
+  const displayItems = React.useMemo(
+    () =>
+      FEEDBACK_CONFIG.filter(item => item.isRejectedItem === isRejected).map(({ id, label, getValue }) => ({
```

Comment:
to double-check: is it correct that we filter by isRejected (which can be both `true` or `false`) instead of filtering by specific value for `item.isRejectedItem`?